docs/Tutorials/Quickstarts/Private-Network-Quickstart.md

```diff
@@ -23,7 +23,7 @@ To run this tutorial, you must have the following installed:
 - [Docker and Docker-compose](https://docs.docker.com/compose/install/) 
 
     !!! important 
-        If using MacOS, enable Docker to use up to 4GB of memory on the [_Advanced_ tab in _Preferences_](https://docs.docker.com/docker-for-mac/).  
+        If using MacOS, enable Docker to use up to 4G of memory or 6G of memory if running Privacy examples. The memory settings are found in the [_Advanced_ tab in _Preferences_](https://docs.docker.com/docker-for-mac/).  
```

Comment:
according to Microsoft Guidelines that we use https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/term-collections/bits-bytes-terms and https://en.wikipedia.org/wiki/Units_of_information#Systematic_multiples the unit should be `GB` and not only the multiplying prefix `G`.

Reply:
I'll update to GB :)